packages/data-explorer-ui/src/theme/common/components.ts

```diff
@@ -214,9 +214,15 @@ export const MuiBreadcrumbs = (theme: Theme): Components["MuiBreadcrumbs"] => {
     styleOverrides: {
       li: {
         ...theme.typography[TEXT_BODY_SMALL_400],
+        margin: 0,
```

Comment:
Updated Breadcrumbs theme to override any CSSBaseline style overrides on `li` and `p` elements.